src/python/grpcio/grpc/_common.py

```diff
@@ -97,3 +98,27 @@ def serialize(message, serializer):
 def deserialize(serialized_message, deserializer):
   return _transform(serialized_message, deserializer,
                     'Exception deserializing message!')
+
+
+class CleanupThread(threading.Thread):
+  """A wrapper around thread for custom join behavior.
```

Comment:
"wrapper around" sounds too much like delegation; what we're doing here is inheritance. How about "A threading.Thread subclass supporting custom behavior at join-time."?